views/main/play-settings.ejs

```diff
@@ -0,0 +1,125 @@
+<% layout('layouts/boilerplate')%>
+<div class="card">
+  <div class="card-box">
+    <div class="image-box">
+      <img
+        class="card-image"
+        id="pixelatedImage"
+        src="<%=previewImage %>"
+        alt="pixelated image"
+      />
+    </div>
+
+    <div id="sessionId" style="display: none"><%= playSessionId %></div>
+    <form
+      action="/send-play-data"
+      method="POST"
+      class="settings-select validated-form"
+      novalidate
+    >
+      <div class="setting">
+        <h5 class="setting-title">Difficulty</h5>
+        <fieldset class="setting-fieldset" name="difficulty">
+          <input
+            class="field-input"
+            type="radio"
+            name="difficulty"
+            value="1"
+            id="1"
```

Comment:
Let's keep label id's clear and understandable, something like `difficulty-1` imo is better